### DISCUSSION TITLE
Add Bookmarks to Rows

### DISCUSSION THREAD

**Discussion by UlrichThiess** [grafana/grafana#45704] [Dashboards, Panels & Visualizations]
Please add Bookmarks (https://www.w3schools.com/html/html_links_bookmarks.asp) to Rows.

If i group panels with rows i could jump into it if i add #rowtitle to the url.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
